ArchiSteamFarm/Bot.cs

```diff
@@ -534,8 +538,17 @@ public sealed class Bot : IAsyncDisposable {
 					continue;
 				}
 
+				ushort maxSetsAllowed = (ushort) (maxItems - result.Count);
+				maxSetsAllowed -= (ushort) (maxSetsAllowed % itemsPerSet);
+				maxSetsAllowed /= itemsPerSet;
+				ushort realSetsToExtract = (ushort) Math.Min(setsToExtract, maxSetsAllowed);
+
+				if (realSetsToExtract == 0) {
+					break;
```

Comment:
I suspect this break doesn't work the way you think it does.

In order to verify that, add a test case proving that input ot 1 card from appID1 and 4 cards from appID2 with sets defined as 2/2, produces expected output of 2 sets from appID2.

Reply:
Not a possibility - As discussed on Discord